radxa/rock-4c-plus/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+let
+  cfg = config.hardware.rockchip;
+in {
+  imports = [
+    ../.
+    ../../rockchip/rk3399
+  ];
+
+  hardware = {
+    radxa.enable = true;
+    rockchip = {
+      enable = true;
+      diskoExtraPostVM = ''
+        dd conv=notrunc,fsync if=${pkgs.ubootRock4CPlus}/idbloader.img of=$out/${cfg.diskoImageName} bs=512 seek=64
```

Comment:
Interesting approach, we can potentially have this also for other rockchip based boards.

Reply:
Yes and this is just a proof-of-concept. Once this PR is accepted I'll add additional devices from us.